src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const nameContent = `My name is ${person.name}`;
```

Comment:
there is no need to create const for ```const nameContent = `My name is ${person.name}`;```
...you can write it directly in the `return ()`